LDAR_Sim/src/default_parameters/g_default.yml

```diff
@@ -15,6 +13,5 @@ start_date: [2017, 1, 1]
 end_date: [2021, 12, 31]
 reference_program: P_OGI
 baseline_program: P_none
-pregenerate_leaks: False
+pregenerate_leaks: True
```

Comment:
It would not, it requires that all sites and leak distributions be the same. I think it should be true, just because the results are easier to interpret because different programs see the same leaks. I will include  infrastructure file check. good call.